src/redux/books/booksSlice.js

```diff
@@ -1,7 +1,26 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  all: [],
+  all: [
+    {
```

Comment:
- [ ] [OPTIONAL] I think it could be much better if you consider using a descriptive array name. This will give you a clear understanding of the code. In addition during debugging, you can easily identify the error ☺️.